config/sevntu_suppressions.xml

```diff
@@ -44,5 +44,6 @@
   <suppress checks="RequireFailForTryCatchInJunit" files="MutableExceptionCheckTest.java"/>
   <suppress checks="RequireFailForTryCatchInJunit" files="CheckstyleAntTaskTest.java"/>
   <suppress checks="RequireFailForTryCatchInJunit" files="AbstractViolationReporterTest.java"/>
+  <suppress checks="RequireFailForTryCatchInJunit" files="AutomaticBeanTest.java"/>
```

Comment:
http://sevntu-checkstyle.github.io/sevntu.checkstyle/#1.39.0 this Check suppose to support Truth, if still a problem please create ticket against this Check.
and place link to here as reason `<!-- until ..... --> `  to know when to remove this suppression.

Reply:
Fix is merged.
Please address this https://github.com/sevntu-checkstyle/sevntu.checkstyle/issues/847#issuecomment-856336514

Reply:
@shashwatj07 please check and and try to eliminate suppressions if `RequireFailForTryCatchInJunit` supports truth fail

Reply:
They can't be eliminated yet https://cloud.drone.io/checkstyle/checkstyle/3902/1/2
sevntu-checkstyle hasn't been released yet

Reply:
just released 1.40.0 just for you, please bump version in this commit.

Reply:
done https://github.com/checkstyle/checkstyle/commit/0d9b3afe600b247eaccf758a0c6ae2bc8409a268

Reply:
removed